CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ All notable changes to this project will be documented in this file.
 * ### NI-Switch
   * #### Added
   * #### Changed
+    * Made init_with_topology a private function.
```

Comment:
I think you can put this under Removed. One thing we should do as well and we haven't is better document the rationale behind some of these decisions. In most cases it's as simple as saying:

> Removed init_with_topology. Clients should use niswitch.Session.__init__(). See #660.

Of course, we need to be good about explaining things in our issues, which is not always the case. Remember this is all open to the world to see.